packages/tools/devtools/devtools-example/jest.config.js

```diff
@@ -2,16 +2,24 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+// Get the test port from the global map and set it in env for this test
+const testTools = require("@fluidframework/test-tools");
+const { name } = require("./package.json");
+
+const mappedPort = testTools.getTestPort(name);
+process.env["PORT"] = mappedPort;
 
 module.exports = {
-	preset: "ts-jest",
-	roots: ["<rootDir>/src"],
+	preset: "jest-puppeteer",
+	globals: {
+		PATH: `http://localhost:${mappedPort}`,
+	},
+	testMatch: ["**/ui-smoke-test/?(*.)+(spec|test).[t]s"],
 	transform: {
-		"^.+\\.tsx?$": "ts-jest",
+		"^.+\\.test.ts?$": "ts-jest",
 	},
-	testRegex: "(/__tests__/.*|(\\.|/)(test|spec))\\.(ts|tsx)?$",
 	testPathIgnorePatterns: ["/node_modules/", "dist"],
-	moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
-	coveragePathIgnorePatterns: ["/node_modules/", "/src/test/"],
-	testEnvironment: "jsdom",
+	// While we still have transitive dependencies on 'uuid<9.0.0', force the CJS entry point:
+	// See: https://stackoverflow.com/questions/73203367/jest-syntaxerror-unexpected-token-export-with-uuid-library
+	moduleNameMapper: { "^uuid$": "uuid" },
```

Comment:
Can't quite tell why the changes here; let's just make sure they're not affecting the existing unit test.

Reply:
Yes. It's required.